### ISSUE TITLE
Blender Importer crashes when loading files converted from .fbx

### ISSUE THREAD

**Issue by MeFisto94** [jMonkeyEngine/jmonkeyengine#447]
Well the title is not really well chosen but here's what I encountered:
I was working on the .fbx -> .blend -> .j3o SDK Shortcut and it turns out that the Blender Importer is incapable of even the most basic files.

What I did:
- Take the usual blender startup file (one box, one light (I deleted the camera)) and export it to .fbx
- Import that .fbx file in Blender and save the .blend (This was done automatically but the process should be the same)
- The file looks OK when viewed in Blender at this point
- Try to load this file and convert it to j3o -> Failure!

```
java.lang.NullPointerException
	at com.jme3.scene.plugins.blender.meshes.MeshHelper.loadVerticesGroups(MeshHelper.java:289)
	at com.jme3.scene.plugins.blender.meshes.TemporalMesh.<init>(TemporalMesh.java:126)
	at com.jme3.scene.plugins.blender.meshes.TemporalMesh.<init>(TemporalMesh.java:100)
	at com.jme3.scene.plugins.blender.meshes.MeshHelper.toTemporalMesh(MeshHelper.java:114)
	at com.jme3.scene.plugins.blender.BlenderLoader.load(BlenderLoader.java:128)
Caused: java.io.IOException: Unexpected importer exception occured: null
	at com.jme3.scene.plugins.blender.BlenderLoader.load(BlenderLoader.java:223)
	at com.jme3.scene.plugins.blender.BlenderLoader.load(BlenderLoader.java:88)
	at com.jme3.asset.DesktopAssetManager.loadLocatedAsset(DesktopAssetManager.java:262)
Caused: com.jme3.asset.AssetLoadException: An exception has occured while loading asset: Models/box.blend
```
Here are the models: 
[box.zip](https://github.com/jMonkeyEngine/jmonkeyengine/files/170206/box.zip)

Oh and it could be that the .fbx importer is simply saving some things strange however that shouldn't raise any NPE :) 

Edit: Further Investigation shows:
```
Start offview panel
Camera created (W: 320, H: 320)
Loading noise constants.
Loading animations that will be later applied to scene features.
Loading ambient light.
Loading blender object.
Type of the object: MESH.
Loading obejct: Cube.001
Importing object of type: MESH
Loading temporal mesh named: Cube.001.
Reading mesh: Cube.001.
Loading vertices and normals from mesh: Cube.001.
Loaded 8 vertices and normals.
Loading vertices colors from mesh: Cube.001.
Loading UV coordinates from mesh: Cube.001.
Loading vertices groups from mesh: Cube.001.
Loading all faces from mesh: Cube.001
Reading BMesh.
Loaded 6 faces.
Loading all edges that do not belong to any face from mesh: Cube.001
Loaded 12 edges.
Loading all points that do not belong to any edge from mesh: Cube.001
Loaded 0 points.
Loading materials.
Loading material.
Loading model's textures.
No textures optimisation applied.
Material's name: Material.001
Reading custom properties.
Child (Cube.001) attached to this node (Cube.001)
Storing loaded feature in blender context and applying markers (those will be removed before the final result is released).
Reading and applying object's modifiers.
Converting temporal mesh into jme geometries.
Converting temporal mesh Cube.001 to jme geometries.
Cube.001 (Node): Child removed.
Preparing faces geometries.
Triangulating temporal mesh.
Triangulating face.
Triangulating face.
Triangulating face.
Triangulating face.
Triangulating face.
Triangulating face.
Appending 6 faces to mesh buffers.
Converting mesh buffers to geometries.
Applying materials to geometries.
Preparing lines geometries.
Child (Cube.0011) attached to this node (Cube.001)
Applying proper scale to the geometries.
Applying animations to the object if such are defined.
Loading constraints connected with this object.
Loading constraints.
Loading custom properties.
Cube.001: (0.0, 2.4013171, 0.0)--> null
Loading blender object.
Type of the object: LAMP.
Loading obejct: Lamp.001
Importing object of type: LAMP
Storing loaded feature in blender context and applying markers (those will be removed before the final result is released).
Reading and applying object's modifiers.
Applying animations to the object if such are defined.
Loading constraints connected with this object.
Loading constraints.
Loading custom properties.
Lamp.001: (4.0762453, 5.903862, -1.005454)--> null
Loading material.
Loading model's textures.
No textures optimisation applied.
Material's name: Material
Only image textures can be loaded as unlinked assets. Generated textures will be applied to an existing object.
Loading temporal mesh named: Cube.
Reading mesh: Cube.
Loading vertices and normals from mesh: Cube.
Loaded 8 vertices and normals.
Loading vertices colors from mesh: Cube.
Loading UV coordinates from mesh: Cube.
Loading vertices groups from mesh: Cube.
Root Node (Node): All children removed.
< Exception here >
Stop offview panel
Component hidden
Root Node (Node): All children removed.
```
So it's actually strange since I didn't find any object called Cube and hence no more structures are on the stack.

**Comment by jmekaelthas** [comment]
I have just mad a commit with fix.
This can happen when an unused mesh block is stored in the file.
Blender stores vertex group names in the parent block of the mesh (the object). So when the importer encountered a mesh to be loaded it couldn't find its parent and read the proper content - this caused NPE to be raised.

**Comment by MeFisto94** [comment]
So this would actually be a bug in Blenders FBX Import, Importing unused vertex groups.
Thanks for the fast fix :) 

Fixed in be3dc06
